content/docs/coldchain/introduction.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Introduction"
+description = "Temperature monitoring and reporting features"
+date = 2023-11-08T15:20:00+00:00
+updated = 2023-11-08T15:20:00+00:00
+draft = false
+weight = 1
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+The cold chain features within open mSupply allow you to:
+- View and manage temperature sensors
+- Import data from Berlinger Q-tag temperature loggers
+- View logs and breach data from sensors
+- Receive notifications of temperature breach events
+ 
+Sensors can be assigned to a storage location, which then allows you to associate temperature logs and breaches with specific stock lines. You can also view historical data to check if your cold chain equipment is operating correctly and to check if your stock might have been affected by temperature variations.
+
+If a temperature breach happens, then a notification is displayed on every page of open mSupply as a warning to check potentially affected stock. Once the breach is dismissed then the notification is no longer shown.
```

Comment:
```suggestion
If a temperature breach happens, then a notification is displayed on every page of Open mSupply as a warning to check potentially affected stock. Once the breach is acknowledged then the notification is no longer shown.
```

Reply:
Also every place where open mSupply is used -> Open mSupply
and cold chain application -> Cold Chain Application (application?)